app/controllers/posts_controller.rb

```diff
@@ -53,6 +53,12 @@ def destroy
     redirect_back_or @post.user
   end
 
+  def update_index
+    @posts = params[:sort_id].to_i == 1 ? load_posts.order_by_lower_price : load_posts.order_by_higher_price
```

Comment:
```suggestion
    @posts = load_posts.order_by_car_price params[:sort_type]
```
Anh gọi thế này là đc